week-1/Homework/mandatory/5-magic-8-ball.js

```diff
@@ -45,10 +45,100 @@ Very doubtful.
 
 // This should log "The ball has shaken!"
 // and return the answer.
-function shakeBall() {}
```

Comment:
It looks like you made a start, but didn't quite finish this one off - can you give it another go? Was there something particular you got stuck on and need a hand with? There's always homework club if you need a little help, or you can ask questions on slack where hopefully people can help out if you're stuck :)